### PR TITLE
update some devtools console docs

### PR DESCRIPTION
**Pull request by kaycebasques**
@pbakaus PTAL

### CODE REVIEW

File: src/content/en/tools/chrome-devtools/debug/console/console-ui.markdown [hunk 1]
```diff
@@ -48,6 +48,8 @@ To open the dedicated **Console** panel, either:
   <kbd>Cmd</kbd>+<kbd>Opt</kbd>+<kbd class="kbd">J</kbd> (Mac).
 * If DevTools is already open, press the **Console** button.
 
+When you open the Console panel, the Console drawer automatically closes.
```
Comment (pbakaus) [inline src/content/en/tools/chrome-devtools/debug/console/console-ui.markdown]: Nit: "closes automatically" sounds a little more natural to me. I also would have used the word "collapse", as the thing is still there at the bottom.

**Comment by pbakaus** [comment]
Awesome work, really like the clearer and simpler wording! One nit that doesn't need to be addressed. LGTM.

File: src/content/en/tools/chrome-devtools/debug/console/console-reference.markdown [hunk 2]
```diff
@@ -35,234 +52,226 @@ Clears the console.
...
+Learn about the functionally equivalent object formatter (`%O`) and more 
+in [String substitution and formatting][of].
 
-Learn about the functional equivalent object formatter (`%O`) and more in [String substitution and formatting](./console-write#string-substitution-and-formatting).
+[of]: console-write#string-substitution-and-formatting
 
 ## console.dirxml(object)
 
-Prints an XML representation of the specified object as it would appear in the Elements panel. For HTML elements, calling this method is equivalent to calling [`console.log()`](#consolelogobject--object-).
+Prints of an XML representation of the descendant elements of `object` if
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/debug/console/console-reference.markdown]: Delete 'of'.

File: src/content/en/tools/chrome-devtools/debug/console/console-reference.markdown [hunk 3]
```diff
@@ -35,234 +52,226 @@ Clears the console.
...
 {% endhighlight %}
 
-`%o` acts either as dir or dirxml depending on the object type (non-DOM or DOM).
+![console.dirxml() example](images/dirxml.png)
 
-## console.error(object [, object, ...])
+## console.error(object [, object, ...]) {#error}
 
-Similar to [`console.log()`](#consolelogobject--object-), `console.error()` additionally includes a stack trace from where the method was called and is styled like an error.
+Prints a message, similar to [`console.log()`](#log), and also styles the
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/debug/console/console-reference.markdown]: Remove the comma.

File: src/content/en/tools/chrome-devtools/debug/console/console-reference.markdown [hunk 4]
```diff
@@ -35,234 +52,226 @@ Clears the console.
...
-console.log("User authenticated");
-console.groupEnd();
+function name(obj) {
+  console.group('name');
+  console.log('first: ', obj.first);
+  console.log('last: ', obj.last);
+  console.groupEnd();
+}
+
+name({"first":"wiley","last":"coyote"});
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/debug/console/console-reference.markdown]: The name is Wile E. Coyete (super genius).

File: src/content/en/tools/chrome-devtools/debug/console/console-reference.markdown [hunk 5]
```diff
@@ -35,234 +52,226 @@ Clears the console.
...
-Closes the logging group that was most recently created with `console.group()` or `console.groupCollapsed()`. See `[console.group()](#consolegroupobject-object)` and `[console.groupCollapsed()](#consolegroupcollapsedobject-object)` for examples.
+Closes a logging group. See [`console.group`](#group) for an example.
 
 ## console.info(object [, object, ...])
 
-This method is identical in function to [`console.log()`](#consolelogobject--object-), but additionally shows an info icon next to the output.
-
-## console.log(object [, object, ...])
+Prints a message like [`console.log()`](#log) but also shows an info icon
+next to the output.
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/debug/console/console-reference.markdown]: You're leaving some things out. I assume the icon you're referring to is a triangle. That's a twisty lets you view the object's members. If any of the members are objects, it has twistys for those too.

**Comment by jpmedley** [comment]
@kaycebasques I found one or two things you should look at.

**Pull request merged**